experiments/AMIP/user_io/leaderboard/compare_with_obs.jl

```diff
@@ -0,0 +1,19 @@
+##########
+# Precip #
+##########
+
+precip_obs = ObsDataSource(;
+    path = "OBS/gpcp.precip.mon.mean.197901-202305.nc",
+    var_name = "precip",
+    # 1 mm/day -> 1 m/s
```

Comment:
```suggestion
    # 1 mm/day -> - 1 kg/m/s2
```
(to be consistent with the latest version of climaatmos)